jdaviz/configs/default/plugins/subset_plugin/tests/test_subset_plugin.py

```diff
@@ -198,6 +199,7 @@ def test_import_spectral_region(cubeviz_helper, spectrum1d_cube, spec_regions, m
     subsets = cubeviz_helper.app.get_subsets()
     assert len(subsets) == len_subsets
     assert len(subsets['Subset 1']) == len_subregions
+    assert cubeviz_helper.app.session.edit_subset_mode.mode == ReplaceMode
```

Comment:
@pllim I added this and the lines below, let me know if that is sufficient coverage.